pkg/config/config.go

```diff
@@ -112,7 +112,7 @@ func LoadIPAMConfig(bytes []byte, envArgs string, extraConfigPaths ...string) (*
 			if err != nil {
 				return nil, "", fmt.Errorf("invalid CIDR %s: %s", n.IPAM.IPRanges[idx].Range, err)
 			}
-			n.IPAM.Range = ipNet.String()
+			n.IPAM.IPRanges[idx].Range = ipNet.String()
```

Comment:
The unit tests should be fixed in the same commit where the production code has changed. 

Doing it like as you're proposing would render tools like `git bisect` useless, and also make things a lot harder for the reviewers.

This commit should be squashed with the commit that added the range configuration to the `IPAMConfig` struct.

Reply:
Done :+1: